frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -33,15 +33,15 @@ Feature: Kiali Overview page
   @overview-page
   Scenario: Filter by namespace
     When user filters "alpha" namespace
-    Then user sees the "alpha" namespace
-    And user doesn't see the "beta" namespace
+    Then user sees the "alpha" namespace card
+    And user doesn't see the "beta" namespace card
 
   @overview-page
   Scenario: Filter by health
     When user filters "Failure" health
     Then user sees the "alpha" namespace
-    And user sees the "beta" namespace
-    And user doesn't see the "default" namespace
+    And user sees the "beta" namespace card
+    And user doesn't see the "default" namespace ca
```

Comment:
ca -> card ?

Reply:
Weird, not sure how I managed to clip chars.